app/models/booking.rb

```diff
@@ -0,0 +1,4 @@
+class Booking < ApplicationRecord
+  belongs_to :user
+  belongs_to :item
```

Comment:
Group decision not to include belongs_to :user through :item - will use booking.item.user instead